RAMP/packer.py

```diff
@@ -53,6 +56,25 @@ def archive(module_path, metadata, archive_name='module.zip'):
     """
     Archives both module and module metadata.
     """
+    dependencies_files = {}
+    for dep_name, dep in metadata['dependencies'].items():
+        if 'local_path' in dep.keys():
+            local_path = eval('f"%s"' % (dep['local_path']), globals())
+            name = os.path.basename(local_path)
+            path = os.path.join('/tmp/', '%s_%s.tgz' % (dep_name, str(uuid.uuid4())))
```

Comment:
Maybe instead we should use tempfile?

Reply:
tempfile will open the file and then we will not be able to open it with `tarfile` to create a targz. Am I missing something?

Reply:
tempfile.mkstemp gives us a guaranteed known temporary file. You can then remove it later with os.unlink(). It's purely to have a knon temporary file.

Reply:
Fixed